docs/relay/mutations.rst

```diff
@@ -28,22 +27,20 @@ subclass of ``relay.ClientIDMutation``.
             faction = get_faction(faction_id)
             return IntroduceShip(ship=ship, faction=faction)
 
-
-
 Accepting Files
 ---------------
 
 Mutations can also accept files, that's how it will work with different integrations:
 
 .. code:: python
 
-    class UploadFile(graphene.ClientIDMutation):
-         class Input:
-             pass
-             # nothing needed for uploading file
+    class UploadFile(ClientIDMutation):
+        class Input:
+            pass
+            # nothing needed for uploading file
```

Comment:
This comment is unrelated to the changes that you did:

It feels weird to have example code with commented out bits. I'd prefer to have a viable example which uses comments to highlight the important things.

Reply:
That's a good conversation to have. We need to start a doc/channel to discuss these things. One thought: Any code we use in the docs that mirrors actual code in the examples, IMHO, ought to be taken as is from the example code. What do you think?

Reply:
Made a note in my local TODO of this.

Reply:
I really like this idea to extract most examples from a working API. I had a few ideas that I brainstormed in this issue:

https://github.com/graphql-python/graphene/issues/970